packages/search-input/src/SearchResultGroup/SearchResultGroup.tsx

```diff
@@ -25,10 +25,12 @@ export const SearchResultGroup = ({
 
   return (
     <div>
+      {/* @ts-expect-error Polymorphic-type mismatch */}
       <InputOption
-        aria-label={label}
+        aria-labelledby={label}
```

Comment:
Looking at the props, `label` is a title string, so it should be `aria-label`.  `aria-labelledby` is for referencing another element.